### PR TITLE
Fixes #160 Favorites list: Column "Simulation" is expendable

### LINKED ISSUE #160: Favorites list: Column "Simulation" is expendable
In the list of favorites of a simulation, the column "Simulation" can be removed, as it always shows the name of the respective simulation. By that, some space for other columns would be made available.

### PR DESCRIPTION
**Pull request by Ian-Du**
Fixes #160 

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please mark relevant options with an `x` in the brackets.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires documentation changes (link at least one [user](https://github.com/Open-Systems-Pharmacology/docs) or [developer](https://github.com/Open-Systems-Pharmacology/developer-docs) documentation issue):
- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
- [ ] Other (please describe):

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Integration tests
- [ ] Unit tests
- [x] Manual tests 
- [ ] No tests required

# Reviewer checklist

Mark everything that needs to be checked before merging the PR.

- [ ] Check if the code is well documented
- [ ] Check if the behavior is what is expected
- [ ] Check if the code is well tested
- [ ] Check if the code is readable and well formatted
- [ ] Additional checks (document below if any)
- [ ] Check if documentation update issue(s) are created if the option `This change requires a documentation update` above is selected

# Screenshots (if appropriate):

# Questions (if appropriate):

### CODE REVIEW

File: src/MoBi.Assets/MoBi.Assets.csproj.orig [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (rwmcintosh) [inline src/MoBi.Assets/MoBi.Assets.csproj.orig:1]: That file should be deleted

File: src/MoBi.BatchTool/MoBi.BatchTool.csproj [hunk 2]
```diff
@@ -74,4 +74,4 @@
     <ProjectReference Include="..\MoBi.UI\MoBi.UI.csproj" />
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+</Project>
```
Comment (rwmcintosh) [inline src/MoBi.BatchTool/MoBi.BatchTool.csproj:77]: Revert this file. It seems like the merge tool has left some artifacts maybe

File: src/MoBi.BatchTool/MoBi.BatchTool.csproj.orig [hunk 3]
```diff
@@ -0,0 +1,81 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (rwmcintosh) [inline src/MoBi.BatchTool/MoBi.BatchTool.csproj.orig:1]: delete

File: src/MoBi.Core/MoBi.Core.csproj.orig [hunk 4]
```diff
@@ -0,0 +1,68 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (rwmcintosh) [inline src/MoBi.Core/MoBi.Core.csproj.orig:1]: I won't keep commenting delete for these files of type ```.orig```. They should all be deleted.

File: src/MoBi/MoBi.csproj [hunk 5]
```diff
@@ -91,4 +91,4 @@
     <Reference Include="Microsoft.VisualBasic" />
     <Reference Include="System.Windows.Forms" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
```
Comment (rwmcintosh) [inline src/MoBi/MoBi.csproj:94]: Same for these merge artifacts. Revert where it's only a new single newline line in a project file.

File: src/MoBi.Presentation/Presenter/EditFavoritesPresenterInSimulation.cs [hunk 6]
```diff
@@ -9,45 +9,51 @@
 
 namespace MoBi.Presentation.Presenter
 {
-   public interface IEditFavoritesInSimulationPresenter : IEditFavoritesPresenter<IMoBiSimulation>, IEditInSimulationPresenter
-   {
-      IEnumerable<IParameter> Favorites();
-   }
+    public interface IEditFavoritesInSimulationPresenter : IEditFavoritesPresenter<IMoBiSimulation>, IEditInSimulationPresenter
```
Comment (rwmcintosh) [inline src/MoBi.Presentation/Presenter/EditFavoritesPresenterInSimulation.cs:12]: There's probably an issue with the spacing in Visual Studio, or there was at the time when this change was made.

Can you do a Resharper cleanup and see if the spacing is fixed?

File: src/MoBi.UI/Views/EditParameterListView.cs [hunk 7]
```diff
@@ -214,8 +214,9 @@ public void SetCaptions(IDictionary<PathElementId, string> captions)
       public void SetVisibility(PathElementId pathElement, bool isVisible)
       {
          _pathBinder.SetVisibility(pathElement, isVisible);
+         _pathBinder.ColumnAt(pathElement).WithShowInColumnChooser(!isVisible);
       }
-
+
```
Comment (rwmcintosh) [inline src/MoBi.UI/Views/EditParameterListView.cs:219]: Here too, there should not be space or tab here.

**Review by msevestre — APPROVED** [review]

**Comment by msevestre** [comment]
@Ian-Du Can you rebase on develop (mostly change of dll). I'd suggest to NOT update the package unless you have to

**Comment by rwmcintosh** [comment]
> @Ian-Du Can you rebase on develop (mostly change of dll). I'd suggest to NOT update the package unless you have to

I merged it. Once the Appveyor is done, I'll squash.

**Pull request merged**